portfolio/src/main/webapp/script.js

```diff
@@ -13,23 +13,26 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Fetches comments from the servers and adds them to the DOM.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function getServerComments() {
+    fetch('/data')
+    .then(response => response.json())
+    .then((myObject) => {
+        const commentsListElement = document.getElementById('comments-container');
+        commentsListElement.innerHTML = '';
+        for(x in myObject) {
+            console.log(myObject[x]);
+            commentsListElement.appendChild(
+                createListElement('Comment: ' + myObject[x]));
+        }
+    });
 }
 
-function getRandomNameUsingArrowFunctions() {
-  fetch('/data').then(response => response.text()).then((name) => {
-    document.getElementById('name-container').innerText = name;
-  });
+/** Creates an <li> element containing text. */
```

Comment:
to match convention, since we are using multiline comment syntax, this can be formatted as:

```java
/** 
 * Creates an <li> element containing text. 
 */

```